src/WidenImage.js

```diff
@@ -0,0 +1,86 @@
+import './WidenImage.css'
+import utilities from './Utilities'
+
+// If enabled, widened images will have thin red dashed border
+const enableDebugBorders = false
+
+const widenAncestors = (el) => {
+  while ((el = el.parentElement) && !el.classList.contains('content_block')) {
```

Comment:
Opps I think I misread your comment - for some reason I wasn't thinking `unit` test. I just added one.